### DISCUSSION TITLE
A new way to track the Arctos Working Group?

### DISCUSSION THREAD

**Discussion by Jegelewicz** [ArctosDB/arctos#4789] [Ideas]
Had a meeting with @jhpoelen today and he showed me how the Open Trait Network is managing their membership:

https://opentraits.org
just recently added - https://opentraits.org/publications

Still uses Github, but the issue template provides more info than a Github profile - tailored to the info the membership might use to communicate with each other. The membership editor uses the information from the github issue to create a page in Markdown, then membership can see who is who:

https://opentraits.org/members.html

**Comment by Jegelewicz** [comment]
@lin-fred @mkoo @ewommack @campmlc

**Comment by campmlc** [comment]
Are you suggesting to use a similar platform, or to join theirs?

**Reply by Jegelewicz** [reply]
A similar plat form

**Comment by lin-fred** [comment]
Anyway for it to link to the Arctos agent page instead so that it can pull names, email and institution from there?

But if not, it's basically an issue per member that gets loaded to markdown?
Would you be able to meet with next week to explain how this all works? @Jegelewicz 

Could this go into the already created arctosdb website or would it have to be its own thing?

Pretty cool though, seems like the best lead we have yet!

**Comment by Jegelewicz** [comment]
> Anyway for it to link to the Arctos agent page instead so that it can pull names, email and institution from there?

That's a question for @dustymc 

> it's basically an issue per member that gets loaded to markdown?

Yep

> Would you be able to meet with next week to explain how this all works? 
Here is how I think it works:
1. User creates an issue
2. Editor copies the stuff from the issue into a markdown page in a special section of the Github repo
3. Someone uses the markdown to create webpages/the "index page" (I think we could copy code from their repo....

> Could this go into the already created arctosdb website or would it have to be its own thing?

It could and I think should go into the arctosdb.org site

**Comment by dustymc** [comment]
There's a shared identifier (github username), it can be used to share data.

Is this an attempt to do something that https://arctos.database.museum/agent/21314876 can't do??

**Comment by jhpoelen** [comment]
There's many ways to manage agents / members etc. of organizations. Arctos has data management features that has evolved over time to meet your needs, Open Traits has a way to do it also. The open traits network (OTN) model came about from a desire for minimizing infrastructure, and we gave github page + jekyll's data/template a go. It's a basic system, and seems to ok work for now. Not quite sure whether I imagine building a collection management system using this current opentraits method to let folks self-manage OTN members, datasets etc. using GitHub workflows (e.g., issues, pull requests, versioning), GitHub pages (hosting web pages generated using Jekyll), and markdown text documents with structured data in it. OTN's data model is relatively simple and the number of records is orders of magnitude smaller than Arctos. It'd be fun experiment . . . and I am sure that there's other fun/important stuff to work on too.

**Comment by Jegelewicz** [comment]
> Is this an attempt to do something that https://arctos.database.museum/agent/21314876 can't do??

Maybe, sorta. Note how the OTN members page makes it very easy to find people near you (via the map) and also to see the organization a person is associated with. New people joining can feel more connected if they can find close or closely associated people. Our page for the AWG is just a big ole list-o-names with no context.

Maybe we can build the thing like OTN has for arctosdb.org using data from the Arctos Agent pages?

**Comment by dustymc** [comment]
>Arctos has data management features

Yep, that's my point - we have the infrastructure, why not use it?

>Note how the ...  and also to ....

Sounds like "just UI" (and relatively straightforward at that), but

>from the Arctos Agent pages

something built from the data using some API (scraper, whatever) also seems reasonable/wouldn't result in people updating one thing and then wondering why that's not in the other/etc.

**Reply by Jegelewicz** [reply]
Yes - I think it would be ideal if we could just use the data from the Agents

**Comment by Jegelewicz** [comment]
Here is my profile on Open Traits - https://opentraits.org/members/teresa-jegelewicz

**Comment by ewommack** [comment]
It is definitely very approachable. I'd like to see how it would look with all of the other data we track for Agents and Members. Or are we thinking this would be a simple who's who page that then would link to the more in-depth Agent page that has all of your objects, identifications, publications, etc?
